spring-integration-core/src/main/java/org/springframework/integration/util/UUIDConverter.java

```diff
@@ -30,15 +32,17 @@
  *
  * @author Dave Syer
  * @author Gary Russell
+ * @author Christian Tzolov
  */
 public class UUIDConverter implements Converter<Object, UUID> {
 
-	public static final String DEFAULT_CHARSET = "UTF-8";
+	public static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
```

Comment:
We cannot do this change since it is a `public` property.
Since it is `final` and really cannot be changed, I suggest the change like:
1. Deprecate this constant: there is really just that `StandardCharsets.UTF_8`. This one in this class is just out of use.
2. Use the mentioned `StandardCharsets.UTF_8` in the code below instead of this constant.